app/domain/authentication/authn_oidc/pkce_support_feature/strategy.rb

```diff
@@ -0,0 +1,43 @@
+module Authentication
+  module AuthnOidc
+    module PkceSupportFeature
+      class Strategy
+        def initialize(
+          authenticator:,
+          client: Authentication::AuthnOidc::PkceSupportFeature::Client,
+          logger: Rails.logger
+        )
+          @authenticator = authenticator
+          @client = client.new(authenticator: authenticator)
+          @logger = logger
+        end
+
+        # Don't love this name...
```

Comment:
Mind either removing this comment, or elaborate on it such that someone other than you (or future you!) could continue the train of thought in an actionable way?